packages/vm/src/errors/vm_error.rs

```diff
@@ -135,46 +135,33 @@ impl From<BackendError> for VmError {
     }
 }
 
-impl From<wasmer_runtime_core::cache::Error> for VmError {
-    fn from(original: wasmer_runtime_core::cache::Error) -> Self {
-        VmError::cache_err(format!("Wasmer cache error: {:?}", original))
+impl From<wasmer::ExportError> for VmError {
+    fn from(original: wasmer::ExportError) -> Self {
+        VmError::resolve_err(format!("Could not get export: {:?}", original))
     }
 }
 
-impl From<wasmer_runtime_core::error::CompileError> for VmError {
-    fn from(original: wasmer_runtime_core::error::CompileError) -> Self {
-        VmError::compile_err(format!("Wasmer compile error: {:?}", original))
+impl From<wasmer_engine::SerializeError> for VmError {
```

Comment:
You can import it from wasmer directly, will make the code less prone to breaking changes in the future: `wasmer::SerializeError`

Reply:
Thanks, done in https://github.com/CosmWasm/cosmwasm/pull/504/commits/971fcc14bb77f3c167817a900297debbb19d70f9